native/packages/react-native-bpk-component-switch/src/BpkSwitch-test.e2e.js

```diff
@@ -20,30 +20,50 @@
 
 /* global element, device, by */
 const sleep = delay => new Promise(_ => setTimeout(_, delay));
+const record = require('./../../../e2e/record');
+const StorybookController = require('./../../../e2e/storybook-controller');
+
+const waitFroAnimationToComplete = async () => sleep(500);
 
 describe('Example', () => {
+  let recordTest;
+
+  beforeAll(async () => {
+    const storybookInterface = new StorybookController('http://localhost:7007');
+    recordTest = record(storybookInterface);
+  });
+
   beforeEach(async () => {
     await device.reloadReactNative();
   });
 
   it('docs:switch', async () => {
-    try {
+    const switchTest = async controllerEmitter => {
+      controllerEmitter.emit('test:started');
```

Comment:
It would be nice if we could get rid of this repition in every testcase also I feel like we should await some common element being avaiable on the screen before we emit `test:started`

Reply:
it's a good shout, but it's definitely not part of this PR, need some proper investigation, where to add that common element etc